pkgs/tools/misc/cinc-client/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, ruby, bundlerApp, bundlerUpdateScript }:
+
+bundlerApp {
+  pname = "chef-bin";
+  gemdir = ./.;
+
+  inherit ruby;
+
+  exes = ["cinc-client"];
+
+  passthru.updateScript = bundlerUpdateScript "chef-bin";
+
+  meta = with lib; {
+    description = "Cinc Client is the community build of the Chef infra client";
```

Comment:
Why are we not packaging the chef infra client?

Reply:
That is a different PR. The difference is licensing. This is effectively a fork, like MyDQL -> MariaDB.